packages/env/lib/config/config.js

```diff
@@ -33,7 +34,7 @@ const md5 = require( '../md5' );
  * Base-level config for any particular environment. (development/tests/etc)
  *
  * @typedef WPServiceConfig
- * @property {?WPSource}                 coreSource    The WordPress installation to load in the environment.
+ * @property {WPSource}                  coreSource    The WordPress installation to load in the environment.
```

Comment:
This is never `null` now that we default to a git source when the user doesn't provide one. As a result, there is probably more code to cleanup, but everything should work correctly in the meantime.